File: In Python/Pascal-Triangle.py
def fact(n):
    f = 1
    for i in range(1, n+1):
        f *= i
    return f

def ncr(n, r):
    return fact(n) // (fact(r) * fact(n - r))

n = int(input("Enter number of rows: "))

for i in range(n):
    print(" " * (n - i), end="")
    for j in range(i + 1):
        print(ncr(i, j), end=" ")
    print()

File: In Swift/Pascal-Triangle.swift
import Foundation

func fact(_ n: Int) -> Int {
    return (1...max(n,1)).reduce(1, *)
}

func ncr(_ n: Int, _ r: Int) -> Int {
    return fact(n) / (fact(r) * fact(n - r))
}

print("Enter number of rows: ", terminator: "")
if let input = readLine(), let n = Int(input) {
    for i in 0..<n {
        print(String(repeating: " ", count: n - i), terminator: "")
        for j in 0...i {
            print("\(ncr(i, j)) ", terminator: "")
        }
        print()
    }
}
